apps/web/app/components/bootscreen/bootscreen.css

```
@@ -0,0 +1,182 @@
+}
+
+.bootscreen-dot:nth-child(2) {
+    animation-delay: 200ms;
+}
+
+.bootscreen-dot:nth-child(3) {
+    animation-delay: 400ms;
+}
+
+/* ---------- Keyframes ---------- */
+@keyframes bsFadeIn {
```

Stylelint (17.3.0)

[error] Expected keyframe name "bsFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)